### DISCUSSION TITLE
Regarding the inlcusion of sidenavbar.

### DISCUSSION THREAD

**Discussion by tejesh01** [twbs/bootstrap#39660] [Ideas]
In bootstrap everything is good but i do not see good side nav bar means if u take a look at your web only u have components like side bar and when it comes small screens it is dynamically hidden and tooggle ,but we this feature in top navbar if it comes to small screens it is automatically toggle means toggle button will be there we need this kind of feature in side nav also pleas include it in a bootstrap same feathre exists in tailwind but i did not feel good view .

**Comment by tejesh01** [comment]
https://flowbite.com/docs/components/sidebar/

please take a look at tailwind side bar

**Comment by tejesh01** [comment]
Hi Any update on abv query??

**Reply by julien-deramond** [reply]
Hi @tejesh01 
I'd say this is something that is partially covered by the Offcanvas component and some examples, but let's think about it later on. We have dot releases to do first right now, and prepare the next minor version with some already prioritized items in the backlog :)
